src/components/Button.tsx

```diff
@@ -70,9 +70,9 @@ export const BackButton: FunctionComponent<BackButtonProps> = ({
   href,
 }: BackButtonProps): JSX.Element => {
   return (
-    <Link href={href}>
-      <a className="govuk-back-link">Back</a>
-    </Link>
+    <a href={href} className="govuk-back-link">
+      Back
+    </a>
```

Comment:
NextJS [<Link>](https://nextjs.org/docs/api-reference/next/link) component was causing a bug with the GOVUK radio/checkbox components whereby expanded checkbox selections weren't displaying when the user clicks "Back" from another page.  Using the <a> element displays exactly the same, and the absence of any optimizations given by NextJS' <Link> aren't discernible (probably because all our pages use SSR!).